src/SprykerEco/Zed/Payone/Dependency/Facade/PayoneToGlossaryFacadeBridge.php

```diff
@@ -32,7 +32,7 @@ public function __construct($glossaryFacade)
      *
      * @return string
      */
-    public function translateByKeyId(int $idKey, array $data = []): string
+    public function translateByKeyId($idKey, array $data = []): string
```

Comment:
Why are we not keeping them?

Reply:
as I was told, a bridge must have the same signature as the bridge method (this is stated in the RFC) but may have the return type defined even if the bridged method does not (not yet in the RFC)

Reply:
RFCs are never relevant here, never.
They get integrated, and the documentation is relevant. Please refer to that one for relevant guidenance.

But again: If there are no technical reasons, we have no need to revert a more stricter user side here IMO.

Reply:
No, there are no technical reasons not to keep the stricter signature here

Reply:
Then we should keep it, as every such change in the future is yet another "major", also cascading out of the dependency.

Reply:
Just to be sure that I got it right. In this PR no existing typehints were removed from bridge method interfaces. In the last commit I only reverted changes that were made earlier (as requested by reviewers)

So it's not that "we are not keeping them", it is rather that "we are not adding them".